build.gradle

```diff
@@ -41,7 +41,7 @@ task coverage(type: JacocoReport) {
 }
 
 dependencies {
-    String jUnitVersion = '5.4.0'
+    String jUnitVersion = '5.10.0'
```

Comment:
Should we revert the jUnitVersion back to 5.4.0?

To me it doesn't affect much but I not too sure from the NUS backend side maybe their marking script might not work?